sycl/test/esimd/dpas.cpp

```diff
@@ -1,19 +1,36 @@
-// RUN: %clangxx -O0 -fsycl -c -Xclang -emit-llvm %s -o %t
-// RUN: %clangxx -O0 -fsycl -c -fsycl-device-only -Xclang -emit-llvm %s -o %t
-// RUN: sycl-post-link -split-esimd -lower-esimd -O0 -S %t -o %t.table
+// RUN: %clangxx -fsycl -c -Xclang -emit-llvm %s -o %t
```

Comment:
why -O0 is removed?

Reply:
I removed it because 
* -O0 did help testing it. 2 and more calls to DPAS still turned/optimized into 1 call. That is why I added those calls to zoo().
* default (-O2) is more typical scenario than -O0. So, let's use it.
* -O0 caused too many mini functions, which grew chances for them to be ordered in randomly.

Reply:
ok, thanks